libs/dh/shared/feature-authorization/src/lib/dh-selected-actor.store.ts

```diff
@@ -57,16 +75,59 @@ export class DhSelectedActorStore extends ComponentStore<SelectedActorState> {
 
             // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
             const actor = r.find((x) => x.id === actorId)!;
+
+            const actors = r
+              .map((x) => ({
+                id: x.id,
+                gln: x.gln,
+                actorName: 'inc.',
+                organizationName: x.organizationName,
+                selected: x.id === actorId,
+              }))
+              .sort((a, b) => {
+                const nameCompare = a.organizationName.localeCompare(
+                  b.organizationName
+                );
+                return nameCompare !== 0
+                  ? nameCompare
+                  : a.gln.localeCompare(b.gln);
+              });
+
+            const actorGroups = Array.from(
+              this.groupBy(actors, 'organizationName').entries()
+            ).map((x) => ({ organizationName: x[0], actors: x[1] }));
+
             return this.patchState({
               isLoading: false,
+              actorGroups: actorGroups,
               selectedActor: {
+                id: actor.id,
                 gln: actor.gln,
+                actorName: 'inc.',
```

Comment:
question: is this going to be part of the API?

Reply:
Jep, actor will get a property called actorName

Reply:
Is it going to be soon? If not, let's add it once the property is available. Also, please take a look at any mock data and whether it should be updated.

Reply:
I'll make backend before completing this PR